RecoLocalCalo/HGCalRecProducers/interface/HGCalCLUEAlgo.h

```diff
@@ -98,6 +101,7 @@ class HGCalCLUEAlgo : public HGCalClusteringAlgoBase {
     iDesc.add<edm::ParameterSetDescription>("doseMap", descNestedNoiseMIP);
     descNestedNoiseMIP.add<double>("noise_MIP", 1. / 100.);
     iDesc.add<edm::ParameterSetDescription>("noiseMip", descNestedNoiseMIP);
+    iDesc.add<bool>("use2x2", true);  // use 2x2 for 3x3 scenario for scint density calculation
```

Comment:
is this supposed to say "2x2 or 3x3" or is it really "for 3x3"?

Reply:
This is a typo. Should be "2x2 or 3x3". Sorry about it.